src/Rules/ThrowsPhpDocRule.php

```diff
@@ -509,6 +516,16 @@ private function filterUnusedExceptions(array $declaredThrows, array $usedThrows
 
 		$unusedThrows = array_diff($unusedThrows, TypeUtils::getDirectClassNames($defaultThrowsType));
 
+		if ($this->allowImplementationUnusedThrows && $functionReflection instanceof MethodReflection) {
+			$declaringClass = $functionReflection->getDeclaringClass();
+			$nativeClassReflection = $declaringClass->getNativeReflection();
+			$nativeMethodReflection = $nativeClassReflection->getMethod($functionReflection->getName());
+
+			if ($nativeMethodReflection->getDocComment() === false) {
+				return [];
```

Comment:
Just an idea: skip when the parent method is abstract and this method is implementation.

Reply:
how about overridden methods?

Reply:
I've implemented your suggestion but also considering method overriding